solidity/test/BondedECDSAKeepFactoryTest.js

```diff
@@ -43,6 +43,7 @@ contract("BondedECDSAKeepFactory", async (accounts) => {
   const notMember = accounts[5]
 
   const keepOwner = accounts[6]
+  const beneficiary = notMember
```

Comment:
Why don't we use `accounts[7]`?

Reply:
No particular reason, just picked an account that not a member. Changed to accounts[7]